fisher.lua

```diff
@@ -421,16 +428,31 @@ function check_incoming_chunk(id, original, modified, injected, blocked)
             if current.monster == false and fish:with('bite_id', current.bite_id) then
                 current.key = original:sub(21)
                 stats.bites = stats.bites + 1
-                local delay = fish:with('bite_id', current.bite_id).delay
-                message(2, 'catching fish in %d seconds':format(delay))
-                windower.send_command('wait %d; lua i fisher catch %d':format(delay, stats.casts))
+                local delay 
+				if settings.random then
+					if math.random() < 0.5 then
+						delay = fish:with('bite_id', current.bite_id).delay + math.random()
+					else
+						delay = fish:with('bite_id', current.bite_id).delay - math.random()
+					end
+				else
+					delay = fish:with('bite_id', current.bite_id).delay
+				end
+				message(2, 'catching fish in %.2f seconds':format(delay))
+				windower.send_command('wait %.2f; lua i fisher catch %d':format(delay, stats.casts))
             elseif current.monster == false and fish:with('bite_id', nil) and settings.fish[tostring(current.bite_id)] == nil then
                 current.key = original:sub(21)
                 stats.bites = stats.bites + 1
                 message(2, 'catching fish when low on time')
             else
-                message(2, 'releasing fish in %d seconds':format(settings.delay.release))
-                windower.send_command('wait %d; lua i fisher release %d':format(settings.delay.release, stats.casts))
```

Comment:
Short circuit logic could be used here again instead of needed the if/else block.